resources/user-agents/apps/1password/1password-generic.json

```diff
@@ -23,14 +23,13 @@
             "iPhone": "iPhone"
           },
           "platforms": [
-            "iOS_A_10_3", "iOS_A_10_2", "iOS_A_10_1", "iOS_A_10_0",
```

Comment:
I removed these because they're here for the UAs in the 3.5 - 4.2 file, which only goes to iOS 8.4.

Reply:
Why did you add `iOS_C_11_x` then?

Reply:
From 4.3 on 1Password changed to use the iOS_C style platform token, even for iPhone, so the iOS_C patterns I added here are to mirror what was added to the 4.3-on file.